fili-core/src/main/java/com/yahoo/bard/webservice/web/handlers/workflow/DruidWorkflow.java

```diff
@@ -85,8 +83,7 @@
     @Inject
     public DruidWorkflow(
             @NotNull DataCache<?> dataCache,
-            @Named("uiDruidWebService") DruidWebService uiWebService,
-            @Named("nonUiDruidWebService") DruidWebService nonUiWebService,
+            @Named("druidWebService") DruidWebService webService,
```

Comment:
We should be able to remove `@Named("druidWebService")` here and the call to `.named("druidWebService")` in `AbstractBinderFactory` since there's only one